ci/logging_tests_computes.yml

```diff
@@ -15,3 +15,26 @@
     - name: "Run Telemetry Logging tests"
       ansible.builtin.import_role:
         name: telemetry_logging
+
+- name: "Verify ctlplane logging containers and files"
+  hosts: computes
+  gather_facts: no
+  ignore_errors: true
+  environment:
+    KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+    PATH: "{{ cifmw_path }}"
+  vars:
+    common_container_test_id: "RHOSO-12753"
+    common_container_list:
+      - ceilometer_agent_compute
+      - ceilometer_agent_ipmi
+      - node_exporter
+
+    common_file_test_id: "RHOSO-12754"
```

Comment:
The file tests for compute are included in: https://github.com/infrawatch/feature-verification-tests/pull/206